cypress/config/production.json

```diff
@@ -0,0 +1,11 @@
+{
+  "baseUrl": "https://giddh.com/login",
+  "pageLoadTimeout" : 80000,
+  "env": {
+    "url": "https://giddh.com/login",
+    "dashBoardUrl" : "https://app.giddh.com/pages/home",
+    "authKey" : "vMvGKUiRYvyZqvwhljopR3R6-5XqBd-JWYu7GmO3xiZp-ySdHroKfx5cxM1KVsrOyz2HaL3DWYL6ko8yofhfsyHdp3a9MHQCJNlWFtdDLe0=",
```

Comment:
Please check if it is safe to put auth key in code

Reply:
This is Giddh Auth key, so there is no issue.